modules/turn/turn.c

```diff
@@ -177,13 +178,17 @@ static bool indication_handler(struct restund_msgctx *ctx, int proto,
 	if (!peer || !data)
 		return true;
 
-	perm = perm_find(al->perms, &peer->v.xor_peer_addr);
+	psa = &peer->v.xor_peer_addr;
+	perm = perm_find(al->perms, psa);
 	if (!perm) {
 		++al->dropc_tx;
 		return true;
 	}
 
-	err = udp_send(al->rel_us, &peer->v.xor_peer_addr, &data->v.data);
+	if (sa_is_loopback(psa) || sa_is_any(psa))
```

Comment:
also sa_is_linklocal?

Reply:
I think just `127.0.0.1` and `0.0.0.0` is fine for now.